msalsas/gdpr-consent-banner-bundle/1.0/config/packages/msalsas_gdpr_consent_banner.yaml

```diff
@@ -0,0 +1,7 @@
+msalsas_gdpr_consent_banner:
+    has_translations: true
+    css: default
+    fade_time: 2
+    text_message: ""
+    accept_message: ""
+    time_to_expire: 30 days
```

Comment:
This looks like your default config. You dont need to expose that in the recipe. Just add one or two values that are likely to change. Like the `time_to_expire`. 

It could also be good with a link to your readme so the users can see what config you have.

Reply:
I know this is not required but I'd prefer to leave them there. Is it a bad practice or has any downside?

Reply:
Friendly ping @Nyholm

Reply:
A recipe is not a replacement for your documentation. 

Please remove them.

Reply:
Done